db/migrate/20161130152326_create_admin_admission_process_images.rb

```diff
@@ -0,0 +1,9 @@
+class CreateAdminAdmissionProcessImages < ActiveRecord::Migration[5.0]
+  def change
+    create_table :admin_admission_process_images do |t|
+      t.references :admission_process, foreign_key: true
```

Comment:
Please add an index, read story carefully again. If you read it, you will get the process of adding index.